### PR TITLE
base_spider: improved title getting

### PR DESCRIPTION
**Pull request by bittirousku**
* Now understands subtitles correctly.

### CODE REVIEW

File: hepcrawl/spiders/base_spider.py [hunk 1]
```diff
@@ -188,12 +200,17 @@ def build_item(self, response):
...
+        if title:
+            record.add_value('title', title)
+        if subtitle:
+            record.add_value('subtitle', subtitle)
         record.add_xpath('date_published', './/dc:date/text()')
         record.add_xpath('source', './/base_dc:collname/text()')
         record.add_value("authors", self.get_authors(node))
         record.add_value('thesis', {'degree_type': 'PhD'})
-        record.add_value('collections', ['HEP', 'THESIS'])  # When it is checked if a thesis is about HEP?
+        # NOTE: When it is checked if a thesis is about HEP:
```
Comment (jalavik) [inline hepcrawl/spiders/base_spider.py]: We can assume that a thesis for INSPIRE is always HEP. However, indeed from base there might be non-hep thesis. We'll have selective harvests combined with automatic classification (e.g. auto-reject) in the workflow on INSPIRE.

**Comment by jalavik** [comment]
Do you have tests for this showing the handling of title and subtitles?

**Comment by bittirousku** [comment]
Added test and modified the loader to take only first element in subtitle list.

**Comment by jalavik** [comment]
Great thanks :+1:

**Pull request merged**